luminoth/utils/image_test.py

```diff
@@ -261,6 +319,126 @@ def testFlipBboxesDiffDtype(self):
             flipped_boxes_float, flipped_boxes_int
         )
 
+    def testRandomPatchImageBboxes(self):
```

Comment:
Add a test where bbox are dropped but some still are returned.
And one to test the `update_condition`.